packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -159,7 +160,7 @@ if (DEV_MODE) {
         currentVersion: Number(functionConfig.pnp_keys.current_version),
         versions: functionConfig.pnp_keys.versions,
       },
-      timeoutMs: Number(process.env.TIMEOUT_MS ?? FULL_NODE_TIMEOUT_IN_MS),
+      fullNodeTimeoutMs: Number(functionConfig.blockchain.timeout_ms ?? FULL_NODE_TIMEOUT_IN_MS),
```

Comment:
looks like you've added this in two places, you can remove it here in favor of where youve defined it in the blockchain config

Reply:
yes exactly